### PR TITLE
fix: fill in ssh key title on setting of repo

### PR DESCRIPTION
**Pull request by appleboy**
related PR: https://github.com/go-gitea/gitea/pull/863

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by plessbd** [comment]
Looking at the code from #683 it looks like it will overwrite a value if it is already set.
```
$("#ssh-key-title").val(arrays[2]);
```

Should this be wrapped as if someone has already put in a value I don't think it should change what the user put in.

Tell me if this should be a new issue and I can change it but this is fixing the code anyway so maybe this could be used for that...

**Comment by tboerger** [comment]
True, @appleboy please add a check if the title is empty before writing to it

**Comment by appleboy** [comment]
@tboerger @plessbd 

https://github.com/go-gitea/gitea/pull/950/commits/c5e88fc374a2d8a55cdfb2fd38665359b4fb1307 Done!

File: public/js/index.js [hunk 1]
```diff
@@ -1493,7 +1493,8 @@ $(function () {
     $("#ssh-key-content").on('change paste keyup',function(){
         var value = $(this).val();
```
Comment (bkcsoft) [inline public/js/index.js]: Is this necessary? Why not `var arrays = $(this).val().split(" ")` ?

Comment (appleboy) [reply public/js/index.js]: @bkcsoft Done

**Comment by bkcsoft** [comment]
LGTM

**Pull request merged**